src/GovUk.Education.ExploreEducationStatistics.Content.Model/Release.cs

```diff
@@ -116,6 +118,7 @@ public IEnumerable<ContentSection> GenericContent
 
                 return Content
                     .Select(join => join.ContentSection)
+                    .Where(cs => cs.Type != ContentSectionType.KeyStatistics)
```

Comment:
I did originally have a TODO here, but ended up removing because I figured you'd get a build failure as soon as `ContentSectionType.KeyStatistics` is removed.

I've readded the comment.